dev/wlp-jakartaee-transform/rules/jakarta-direct.properties

```diff
@@ -96,16 +96,47 @@ com.sun.xml.bind.defaultNamespaceRemap=org.glassfish.jaxb.defaultNamespaceRemap
 com.sun.xml.internal.bind.defaultNamespaceRemap=org.glassfish.jaxb.defaultNamespaceRemap
 
 http\://java.sun.com/xml/ns/jaxb=https://jakarta.ee/xml/ns/jaxb
+http\://java.sun.com/xml/ns/jaxws=https://jakarta.ee/xml/ns/jaxws
```

Comment:
These were properties and a namespace that I found through searching the CXF code that we use with xmlws-3.0. They have no relevance to the tooling changes per say, but will be needed anyway.

Reply:
You cannot do this one.  I had done it originally, but I removed it because it will cause us not to be able to read older xml files for all of the code.  It should not be needed to have it.  If you think it is still needed please let me know where.

Reply:
excellent